schema.json

```diff
@@ -0,0 +1,514 @@
+{
+  "$schema": "http://json-schema.org/LivingDocumentation",
+  "title": "Intermediary JSON for LivingDocumetation",
+  "type": "array",
+  "items": {
+    "type": "object",
+    "properties": {
+      "FullName": {
+        "type": "string"
+      },
+      "Type": {
+        "type": "number"
+      },
+      "Modifiers": {
+        "type": "number"
+      },
+      "BaseTypes": {
+        "type": "array",
+        "items": {
+          "type": "string"
+        }
+      },
+      "Fields": {
+        "$ref": "#/$defs/fields"
+      },
+      "Constructors": {
+        "$ref": "#/$defs/constructors"
+      },
+      "Methods": {
+        "$ref": "#/$defs/methods"
+      },
+      "Properties": {
+        "$ref": "#/$defs/$properties"
+      },
+      "Attributes": {
+        "$ref": "#/$defs/attributes"
+      },
+      "EnumMembers": {
+        "$ref": "#/$defs/enumMembers"
+      },
+      "Events": {
+        "$ref": "#/$defs/events"
+      },
+      "DocumentationComments": {
+        "$ref": "#/$defs/documentationComments"
+      }
+    },
+    "additionalProperties": false,
```

Comment:
Given that we (I!) have since learned that actually the C# implementation is actually fine with additional properties, do we still want to ban them here?

Reply:
I would say yes, just to limit how easily things can be added (and become a mess), but maybe this is something we should discuss with the rest of the team?

Reply:
I would say it's up to you - so let's keep it.